commit 3e7a7df183a3fea024376797dd83e955ef9851c5
Author: Paul Davis <paul@linuxaudiosystems.com>
Date:   Fri Oct 4 17:51:03 2013 -0400

    fix startup errors with 2.x related to boost linkage on some platforms
    
    Note: this should not be required because we do not actually link with ANY boost libraries, so the
    occurence of this bug in ardour at run time indicates that the wrong version of boost was used and/or
    incorrect flags for using boost are in use.
    
    Fix from  David Henningsson <david.henningsson@canonical.com>

diff --git a/libs/surfaces/generic_midi/SConscript b/libs/surfaces/generic_midi/SConscript
index 27f8865..ce37359 100644
--- a/libs/surfaces/generic_midi/SConscript
+++ b/libs/surfaces/generic_midi/SConscript
@@ -42,7 +42,8 @@ genericmidi.Merge ([
 	libraries['usb'],
 	libraries['xml'],
 	libraries['glib2'],
-	libraries['glibmm2']
+	libraries['glibmm2'],
+	libraries['boost']
 	])
 
 libardour_genericmidi = genericmidi.SharedLibrary('ardour_genericmidi', genericmidi_files)
diff --git a/libs/surfaces/tranzport/SConscript b/libs/surfaces/tranzport/SConscript
index c2ab351..dae2ba2 100644
--- a/libs/surfaces/tranzport/SConscript
+++ b/libs/surfaces/tranzport/SConscript
@@ -63,6 +63,7 @@ tranzport.Merge ([
     libraries['xml'],
     libraries['glib2'],
     libraries['glibmm2'],
+    libraries['boost'],
     libraries['usb'],
     libraries['sndfile']
     ])
